news/_posts/2024-02-24-open-door.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: news
+title: "Lab members participate in ASU Open Door 2024"
+author: "Navish Wadhwa"
```

Comment:
@shabduli, since you wrote this news piece, you should be the author. Can you change that?